schemas/exchange/min-meta/json-min-db-pdbx_core-col-pdbx_core_entry.json

```diff
@@ -763,6 +763,174 @@
+                        "value": "AlphaFoldDB",
+                        "detail": "                                 AlphaFoldDB"
+                     },
+                     {
+                        "value": "BMRB",
+                        "detail": "                                 Biological Magnetic Reference Data Bank"
+                     },
+                     {
+                        "value": "EBI",
+                        "detail": "                                 European Bioinformatics Institute"
+                     },
+                     {
+                        "value": "EMDB",
+                        "detail": "                                 Electron Microscopy Data Bank"
+                     },
+                     {
+                        "value": "MODBASE",
+                        "detail": "                                 Database of Comparative Protein Structure Models"
+                     },
+                     {
+                        "value": "ModelArchive",
+                        "detail": "                                 ModelArchive"
+                     },
+                     {
+                        "value": "NDB",
+                        "detail": "                                 Nucleic Acid Database"
+                     },
+                     {
+                        "value": "PDB",
+                        "detail": "                                 Protein Data Bank"
+                     },
+                     {
+                        "value": "PDB-Dev",
+                        "detail": "                                 PDB-Dev integrative/hybrid methods"
+                     },
+                     {
+                        "value": "PDBE",
+                        "detail": "                                 Protein Data Bank Europe"
+                     },
+                     {
+                        "value": "PDB_ACC",
+                        "detail": "                                 Protein Data Bank Versioned Accession"
+                     },
+                     {
+                        "value": "RCSB",
+                        "detail": "                                 Research Collaboratory for Structural Bioinformatics"
+                     },
+                     {
+                        "value": "SWISS-MODEL_REPOSITORY",
+                        "detail": "                                 Swiss-Model Repository"
+                     },
+                     {
+                        "value": "WWPDB",
+                        "detail": "                                 Worldwide Protein Data Bank"
+                     }
+                  ],
+                  "rcsb_description": [
+                     {
+                        "text": "An abbreviation that identifies the database.",
+                        "context": "dictionary"
+                     }
+                  ]
+               },
+               "pdbx_DOI": {
```

Comment:
Do we want to load the DOI as well? I think we're currently building DOIs on the fly in frontend

Reply:
Good point @valasatava. 

It is an attribute in the public `database_2` category. Is it ok to have it in the data warehouse for external users?

Reply:
@valasatava do you recommend that we suppress `pdbx_DOI` explicitly? Or can it be included as is?

Reply:
There is no issue with including this item, but I am being cautious about what we expose in the public API. @josemduarte what's your opinion?

Reply:
I think it can't hurt to include it. So that we offer the data out to users. But let's not add to search.

Reply:
Thanks @josemduarte. There is no search metadata for `pdbx_DOI`.